src/core/resources/internal-registry.ts

```
@@ -0,0 +1,493 @@
+
+/**
+ * Union type for all internal resource configurations
+ */
+export type InternalResourceConfig = FileSystemResourceConfig;
+
+/**
+ * Services available to internal resource handlers
+ */
+export interface InternalResourceServices {
+    // Add services as needed for resource implementations
+}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)